index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link id="birdtab" rel="shortcut icon" type="image/ico" href="./images/favicon.ico">
+    <title>Twitter. It's what's happening.</title>
+    <script src="./index.js"></script>
+    <link href="./master.css" rel="stylesheet">
+    
+</head>
+
+<body>
+    <main class="container">
+       <section class="blue">blue
+           <ul>
+               <li class="li1"  </li >Follow your interests</li>
+               <li class="li2" >Hear what people are talking about</li>
+               <li class="li3">Join the conversation</li>    
+            </ul>
+      </section>
+        <section class="white">white
+            <form>
+                <input type="text" placeholder="Phone, email, or username ">
+                <input type="text" placeholder="Password">
+                <button>login</button> 
+                <h1>See what’s happening in the world right now</h1>
+                <h2>Join Twitter today.</h2>
+                <button>Sign up</button>
+                <button>Log in</button>
+
+            </form>
+        </section>
+        <section class="footer">footer
```

Comment:
You probably meant to have a div `<footer>` . Right now on your page it literally says "footer" .